htdocs/core/class/extrafields.class.php

```diff
@@ -268,13 +270,13 @@ public function updateExtraField($attrname, $label, $type, $pos, $size, $element
 		// Create field into database except for separator type which is not stored in database
 		if ($type != 'separate') {
 			dol_syslog(get_class($this).'::thisupdate', LOG_DEBUG);
-			$result = $this->update($attrname, $label, $type, $size, $elementtype, $unique, $required, $pos, $param, $alwayseditable, $perms, $list, $help, $default_value, $computed, $entity, $langfile, $enabled, $totalizable, $printable, $moreparams);
+			$result = $this->update($attrname, $label, $type, $size, $elementtype, $unique, $required, $pos, $param, $alwayseditable, $perms, $list, $help, $default_value, $computed, $entity, $langfile, $enabled, $totalizable, $printable, $moreparams, $emptyonclone);
```

Comment:
The parameter emptyonclone is at position of the other parameter aiprompt.

Reply:
Ok I fixed it. Don't know why "aiprompt" is here but ok.